.github/workflows/reduce-readme.yml

```diff
@@ -0,0 +1,34 @@
+name: Reduce README
```

Comment:
Rename to "WildFly Quickstarts Reduce READMEs"

Reply:
done